study-builder/tenants/cmi/tenant.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  - name: Register User in Pepper
+    script: ../register-user-in-pepper.js
+    stage: login_success
+    enabled: true
+    order: 1
+
+pages:
+  - name: login
+    html: ./pages/login.html
+    enabled: true
+  - name: password_reset
+    html: ./pages/password_reset.html
+    enabled: true
+
+emailTemplates:
+  - template: reset_email
+    body: ./emails/change_password.html
+    from: Count Me In <info@joincountmein.org>
+    resultUrl: '##SERVER_BASE_URL##/pepper/v1/post-password-reset?clientId={{ application.clientID }}'
+    subject: >-
+      {% if client.client_metadata.study == "ANGIO" %} Angiosarcoma Project
+      {% elsif client.client_metadata.study == "cmi-brain" %} Brain Cancer Project
+      {% elsif client.client_metadata.study == "cmi-mbc" %} Metastatic Breast Cancer Project
+      {% elsif client.client_metadata.study == "CMI-OSTEO" %} Osteosarcoma Project
+      {% endif %} Password Reset
+    urlLifetimeInSeconds: 432000
+    syntax: liquid
+    enabled: true
+
+# NOTE: client names must match what's in Auth0's dashboard, otherwise deploy
+# will create new clients. Everytime deploy happens, client configurations here
+# will overwrite what's in Auth0. If there are additional urls not in here
+# (e.g. localhost callback urls) then those need to be added back manually
+# afterwards. Other settings not configured here will also need to be
+# configured manually.
+clients:
```

Comment:
The client names don't match up so this will create new clients. Try not to run this as-is, unless that's what you want, and if you did please cleanup after yourself :) Once this is merged, I'll rename the clients so they match up.